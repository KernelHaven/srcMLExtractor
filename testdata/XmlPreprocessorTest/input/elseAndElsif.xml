<unit language="C">
	<cpp:if>#<cpp:directive>if</cpp:directive><expr>A</expr></cpp:if>
	<empty_stmt>;</empty_stmt>
	<cpp:elif>#<cpp:directive>elif</cpp:directive><expr>B</expr></cpp:elif>
	<empty_stmt>;</empty_stmt>
	<cpp:else>#<cpp:directive>else</cpp:directive></cpp:else>
	<empty_stmt>;</empty_stmt>
	<cpp:endif>#<cpp:directive>endif</cpp:directive></cpp:endif>
</unit>
