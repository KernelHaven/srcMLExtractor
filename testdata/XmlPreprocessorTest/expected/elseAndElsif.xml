<unit language="C">
	<cpp:if>#<cpp:directive>if</cpp:directive><expr>A</expr>
		<empty_stmt>;</empty_stmt></cpp:if>
	<cpp:elif>#<cpp:directive>elif</cpp:directive><expr>B</expr>
		<empty_stmt>;</empty_stmt></cpp:elif>
	<cpp:else>#<cpp:directive>else</cpp:directive>
		<empty_stmt>;</empty_stmt>
	</cpp:else>
</unit>
